libs/eo/auth/domain/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
```

Comment:
Do we really need all this ? Seems like waste of E-Ink :P

Reply:
I'm not sure if we still have too, however from the beginning we had and it's added automatically by our CI pipeline.